algorithm_sort.rb

```diff
@@ -132,3 +132,30 @@ def sort(words)
 end
 
 sort(words)
+
+# sorts words in the array based on their length
+
+def sort_length(words)
+  # initializes the array that will store the ordered list of words
+  ordered_array = []
+  # initializes the hash used to store the words and their length
+  collection = {}
+
+  # maps each word to its length in the hash
+  words.each do |word|
+    collection[word] = word.length
+  end
+
+  while collection.empty? == false
+    # stores the length of the word in the hash with the minimum number of letters
+    temp = collection.values.min
```

Comment:
Great use of the `.min` method on the `collection` hash! That method call is new to me.